api/lib/getJSON.js

```diff
@@ -45,6 +68,24 @@ const prepVerse = (row, includeMeta = false, liveSearch = 0) => {
     verse.lineNo = row.LineNo;
     verse.updated = row.Updated;
     verse.visraam = JSON.parse(row.Visraam);
+    /* eslint-disable no-param-reassign */
+    if (Array.isArray(verse.visraam)) {
```

Comment:
would prefer this to be in a seperate function that you call, makes it more readable imo (ie if the function name was `compactVishraams` or something, its more declarative so more easier to understand when reading, ofc imo)